release.yaml

```diff
@@ -59,6 +59,7 @@ rancher-eks-operator-crd:
 - 100.0.2+up1.1.3-rc2
 rancher-aks-operator:
 - 100.0.3+up1.0.4-rc1
```

Comment:
why is rc1 still in here?

Reply:
It is correct for the old RC versions to be in the release.yaml file.

Reply:
@thedadams Ok thanks!